### PR TITLE
[TAN-646][TAN-769]Treat /live and /ready as the others and don't log them.

### PR DESCRIPTION
**Pull request by martinBlacklane**

### CODE REVIEW

File: lib/kiev/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Kiev
-  VERSION = "4.8.3"
+  VERSION = "4.8.4"
```
Comment (pbauzyte) [inline lib/kiev/version.rb:4]: is it a patch?

Comment (martinBlacklane) [reply lib/kiev/version.rb:4]: it is a backwards compatible extension. I am not aware of our guidelines here, should we rather increase to 4.9.0?

Comment (thiagopio) [reply lib/kiev/version.rb:4]: It seems to be a minor version change. I use https://semver.org/ as reference

Comment (pbauzyte) [reply lib/kiev/version.rb:4]: to be fair, I have no idea of Blacklane guidlines, but as @thiagopio , I was thinking maybe a minor to 4.9.0?

Comment (martinBlacklane) [reply lib/kiev/version.rb]: Changed.

**Review by thiagopio — APPROVED** [review]

**Comment by thiagopio** [comment]
https://github.com/blacklane/kiev/actions/runs/3409522465/jobs/5672128919

The header `Content-Type` somehow is in lower case and fails during assertion.

**Review by pbauzyte — APPROVED** [review]

**Pull request merged**